labs/sysdev-kernel-cross-compiling/sysdev-kernel-cross-compiling.tex

```diff
@@ -64,17 +64,28 @@ \section{Cross-compiling environment setup}
 Also, don't forget to either:
 
 \begin{itemize}
-\item Define the value of the \code{ARCH} and \code{CROSS_COMPILE}
+\ifdefstring{\labboard}{beaglebone}{
+  \item Define the value of the \code{ARCH=arm} and \code{CROSS_COMPILE=arm-training-linux-musleabihf-}
```

Comment:
It is intentional that we don't give directly the values, we want people to think and find out what are the correct values.

Reply:
I see, it makes sense.